sdk/eventhub/event-hubs/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 - Fixed a potential issue with deadlock where greedy consumers could
   starve out other consumers, preventing us from properly balancing.
+- Updated to use the latest version of the `@azure/core-amqp` package.
```

Comment:
> Updated to use the latest version of the `@azure/core-amqp` package which in turn uses latest version of the `rhea` package.

Perhaps add a link to the issue/PR in rhea for "more details"

Reply:
Is that something the user should care about? From their point of view just updating core-amqp is enough to have this affect. I could add it to the core-amqp change log

Reply:
I look at it as a cherry on top, appealing to the engineer in the reader. When I see something like `This improved our bundling approach`, my next thought would, `Oooh, I wonder what actual change resulted in this win` :)